src/theme/_horizontal-line.css

```diff
@@ -1,19 +1,23 @@
 /* stylelint-disable selector-max-compound-selectors,
    selector-combinator-space-before, indentation,
-   declaration-no-important */
+   declaration-no-important, max-line-length */
 
 .ck-content hr {
-  margin: var(--ck-spacing-standard) 0;
-  background-color: #d8dcde; /* GREY-300 */
+  margin: 0;
+  background-color: $ZD-PALETTE-GREY-300;
   height: 1px;
 }
 
 .ck .ck-widget.ck-horizontal-line {
+  padding: var(--ck-spacing-medium) 0;
+  margin-block-start: calc(var(--ck-spacing-medium) * -1);
+  margin-block-end: calc(var(--ck-spacing-medium) * -1);
```

Comment:
Since there doesn't appear to be RTL differentiation, would it be better to use cross-browser compatible shorthand?

`margin: 0 calc(var(--ck-spacing-medium) * -1);`

Reply:
`margin-block` allows the margins to collapse against each other, which is used throughout the other typography elements.

https://caniuse.com/mdn-css_properties_margin-block-start we should be good on browser support since CK doesn't support IE11.